docker/kafka-setup/Dockerfile

```diff
@@ -1,12 +1,10 @@
 # This "container" is a workaround to pre-create topics
 FROM confluentinc/cp-kafka:5.4.0
 
-ENV METADATA_AUDIT_EVENT_NAME = "MetadataAuditEvent_v4"
-ENV METADATA_CHANGE_EVENT_NAME = "MetadataChangeEvent_v4"
-ENV FAILED_METADATA_CHANGE_EVENT_NAME = "FailedMetadataChangeEvent_v4"
+ENV METADATA_AUDIT_EVENT_NAME="MetadataAuditEvent_v4"
```

Comment:
Sorry about this error. PR to fix this recently got pushed! Can you merge latest master?